applications/DamApplication/custom_elements/solid_element.cpp

```diff
@@ -0,0 +1,2426 @@
+//
+//   Project Name:        KratosDamApplication $
+//   Created by:          $Author:            JMCarbonell $
+//   Last modified by:    $Co-Author:                     $
+//   Date:                $Date:                July 2013 $
+//   Revision:            $Revision:                  0.0 $
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_elements/solid_element.hpp"
+#include "dam_application_variables.h"
+
+
+namespace Kratos
+{
+
+/**
+ * Flags related to the element computation
+ */
+KRATOS_CREATE_LOCAL_FLAG( SolidElement, COMPUTE_RHS_VECTOR,                 0 );
+KRATOS_CREATE_LOCAL_FLAG( SolidElement, COMPUTE_LHS_MATRIX,                 1 );
+KRATOS_CREATE_LOCAL_FLAG( SolidElement, FINALIZED_STEP,                     2 );
+
+
+//******************************CONSTRUCTOR*******************************************
+//************************************************************************************
+
+SolidElement::SolidElement( )
```

Comment:
Have you checked if the `SmallDisplacement` element can be used instead of copying this one?

Reply:
Thanks @rubenzorrilla I adressed that. not only elements, but some CL too

Reply:
For the CLs I did the same as with the elements